packages/auth-plugins/src/Login.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { RandomAreaPlotAnimation } from '@deephaven/components';
+import Logo from './logo.png';
```

Comment:
It is something we do with DHE, which has some server side configuration: https://github.com/deephaven-ent/iris/blob/d6950c95398cbe9a400eca8ba90e11201dcdae7c/web/server-frontend/src/main/java/com/illumon/iris/web/server/AbstractIrisWebServer.java#L170
We don't have that same configuration on the DHC side.

That being said, I'll wire it up the same on the client. Just means the logo is in 3 public paths (ide, embed-grid, embed-chart) instead of just being packaged with auth plugins.

And yes, a custom auth plugin would be able to customize whatever.